components/LocaleSelector.vue

```diff
@@ -2,7 +2,7 @@
   <div class="w-24">
     <select
       v-model="selectedLocale"
-      class="rounded-full px-2 py-1 border-2 border-primary/80 drop-shadow-md text-primary-text bg-secondary-bg/5 hover:text-primary-hover transition-colors"
+      class="rounded-full md:w-full px-2 py-1 md:px-1 md:py-1.5 border-2 border-primary/80 md:border-primary/60  drop-shadow-md text-primary-text bg-secondary-bg/5 md:bg-primary-bg hover:bg-primary-hover/10 transition-colors"
```

Comment:
I need to add documentation around this, but we aren't using `md:` anymore in the project. Instead use `portrait:` or `landscape:`. 
This let's us design around device orientation and we don't have to have custom resolutions for every new device  that comes out.

Reply:
@ermish should I be writing it as if portrait is the default? Or landscape is the default?

Reply:
that's a fair question we haven't decided yet 🤔 
we've been building as desktop mode first, but maybe we should default the mobile view....we don't know what our user base is yet. 
@theyokohamalife you up for mobile first from now on? (obviously we need to add documentation around this eventually)

Reply:
Just my two cents for @ermish and @theyokohamalife before you make the decision. The project uses tailwind which in the documentation is [mobile first](https://tailwindcss.com/docs/responsive-design#working-mobile-first ). So I would suggest that we default to portrait and then change to landscape. Just due to the css preprocessor we are using and the way it was designed.

Reply:
sounds good to me! @theyokohamalife ?